routers/posts.py

```
@@ -1,29 +1,278 @@
+        "pagination": {
+            "page": page,
+            "limit": limit,
+            "total": total,
+        }
+    }
 @router.post("", status_code=status.HTTP_201_CREATED)
-def create_post(data: PostCreate):
-    return {"message": "게시글 작성"}
+def create_post(
+    data: PostCreate,
+    current_user: dict = Depends(get_current_user)
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)